lib/utility.js

```diff
@@ -2,9 +2,9 @@ const XLSX = require('xlsx')
 const { jwtSession, newParser } = require('@propelsoftwaresolutions/propel-sfdc-connect')
 
 function convertDataByType(data, type) {
-  let returnData = data
+  let returnData = Buffer.from(data, 'base64')
```

Comment:
this breaks the normal csv flow as now `returnData` is in base64. That is why I changed it.

Reply:
CSV data is also sending ad base64 from Apex or is there test I need to fix?

Reply:
yeah I happened to look at QueueService and saw your update to send base64 encoded data.
So we need to keep in encoding from salesforce, I think. Seems more proper to do it that way.

Update this PR to convert the returnData to plan text at the return not inside the `.xlsx` conversion.

Reply:
a bit lost. what did you mean by the last line? 

Both paths should be csv data at this point `return newParser(returnData)`. Or you mean don't use `let returnData` for both?

Reply:
also we need to update the postman scripts to accept base64 encoded files.

Reply:
so the files expect a string, and since we now pass in a binary you only convert to a string if the file is of type `.xlsx` so if the file is type `.csv` it stays a binary and is passed to the processing file.

Reply:
Good catch. So I test around the `newParser` and it takes both. that's why my manual test all works. 

I'll still convert it to string to be consistent.